### ISSUE TITLE
диалектные падежи на -Д

### ISSUE THREAD

**Issue by veramal** [tuffnatty/suddenly#221]
У _пазыды_ есть в т.ч. такой разбор:
n пас ‛голова’
--------------(з)Ы-дЫ------
--------------3pos-AccDial------

У диалектного варианта Acc _дЫ_ не было записано _н_ в скобках, как везде в подобных случаях - дописала.   Видимо, потому, что примеров сочетание _дЫ_ с 3 лицом посессива пока не встретилось - но такая цепочка, как сейчас, точно невозможна. Вообще после носовых _ды_ бывает, в т.ч. при посессорах 1-2 лица, вот примеры из корпуса:  _паламды_ 'моего ребенка', _палаңды_ 'моего ребенка', _ӧрекенді_ ‘старушку’.

**Comment by veramal** [comment]
Понятно, почему _пазыды_ в смысле аккузатива не говорят - ведь в 3 лице другой аккузатив, единственная возможная форма _пазын_. Так что стерла обратно (н), написала правило 37, запрещающее 3pos+AccDial.

**Comment by veramal** [comment]
Дописала в правило 37 запрет AccDial дЫ присоединяться к словам с пометой poss, если у них нет посессивного показателя. Иначе имеем такой разбор:
алынды
n алын <+poss> ‛перёд’ алн
---------------дЫ------
---------------AccDial-----

**Comment by adybo** [comment]
Да, это из переделанной мудраковской таблицы — морфонологич. перестройка. Т.е.бывает, что там вообще морфонема Д а не Н, а бывает, что Н, но с другими условиями — после гласной д, а после носовой н

  
>Вторник, 14 марта 2023, 19:37 +03:00 от veramal ***@***.***>:
> 
> 
>У  пазыды есть в т.ч. такой разбор:
>n пас ‛голова’
>--------------(з)Ы-дЫ------
>--------------3pos-AccDial------
>У диалектного варианта Acc  дЫ не было записано  н в скобках, как везде в подобных случаях - дописала. Видимо, потому, что примеров пока не встретилось - но такая цепочка, как сейчас, точно невозможна. Вообще после носовых  ды бывает, в т.ч. при посессорах 1-2 лица, вот примеры из корпуса:  паламды 'моего ребенка',  палаңды 'моего ребенка',  ӧрекенді ‘старушку’.
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you are subscribed to this thread. Message ID:  <tuffnatty/suddenly/issues/221 @ github . com> 
 
 
--
Anna Dybo
Отправлено из Почты  Mail.ru

**Comment by veramal** [comment]
По условиям у нас так: морфонема Н включает только варианты *н* и *т*,
морфонема Д - варианты *д*, *н* и *т*. Из-за этого сейчас после глухих и
после носовых всегда получается омонимия разборов на ровном месте: через
Д-морфонему и через Н-морфонему (см., допустим, *хамны* и *хосты*). Может,
все же оставить для диалектных лексем морфонему не Д, а *д*? Пусть она
будет выделяться только в тех случаях, когда аккузатив/генитив начинаются с
*д*. А в прочих случаях пусть разбирается одним способом.

**Comment by adybo** [comment]
Да, согласна

**Comment by veramal** [comment]
Переписала в таблице диалектные варианты, теперь они имеют такой вид: Gen дЫӊ, Acc дЫ, Gen.3pos ди. Ни одной причины иметь отдельный посессивный показатель аккузатива дЫ или ДЫ не вижу - он встречается только в неправильных разборах типа _пазыды_. Убрала дЫ из посессивных диалектных показателей, тепреь там дырка.

**Comment by veramal** [comment]
Правило 37 тем самым не нужно, зачеркнула.

**Comment by adybo** [comment]
Да, согласна

**Comment by veramal** [comment]
Пазыды по-прежнему разбирается через посс+асс.

**Comment by tuffnatty** [comment]
> Пазыды по-прежнему разбирается через посс+асс.

А почему не должно (после удаления правила 37)?

**Comment by veramal** [comment]
Потому что в наборе посессивных окончаний варианта дЫ быть не должно ("Убрала дЫ из посессивных диалектных показателей, теперь там дырка.")

**Comment by tuffnatty** [comment]
а где можно найти этот набор?

**Comment by tuffnatty** [comment]
я не тормоз. Нашел. Убрал. Но тогда перестает разбираться _никтерімді_

**Comment by veramal** [comment]
Где и все остальное - в документе, таблица на стр. 1-2.

**Comment by veramal** [comment]
А, забыла про такую возможность. Тогда нужно оставить показатель и вернуть правило 37. В документе исправила.

**Comment by veramal** [comment]
Кажется, можно это сформулировать проще: Диалектный посессивный показатель аккузатива Acc дЫ возможен только при 1sg.poss или 2sg.poss.

**Comment by veramal** [comment]
Пазыды, алынды - разбора с аккузативом сейчас нет, правильные разборы остались.
